src/app/modules/item/http-services/get-item-by-id.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
```

Comment:
I think there's an issue, I created a hidden task visible only to task owner, and when watching a group or a user it says "[Group/User] can view this activity".
I think the permissions taken to read that are `item.permissions` instead of `item.watchedGroup.permissions`
To reproduce:
1. As usual user, go to [this user page](https://dev.algorea.org/branch/feature/observation-mode-display-group-permission-on-item/en/#/groups/users/752024252804317630)
2. Watch, then navigate to "Activities > Parcours Officiels > Motif Art > Thomas Tasks > Locked content"
3. Then you'll see:
![image](https://user-images.githubusercontent.com/16257159/148440343-627be1c4-2e49-40a1-85e3-b5717f51a381.png)

But according to the api:
![image](https://user-images.githubusercontent.com/16257159/148440395-7ccd8f3f-bc72-433e-9315-c7f204abc07b.png)

(See [itemView](https://france-ioi.github.io/algorea-devdoc/api/#operation/itemView) operation for more info)

Reply:
Fixed